docs/2.guide/2.directory-structure/1.components.md

```diff
@@ -336,6 +336,11 @@ The content will not be included in production builds and tree-shaken.
     <DevOnly>
       <!-- this component will only be rendered during development -->
       <LazyDebugBar />
+
+      <!-- if you ever require to have a replacement during production -->
+      <template #fallback>
```

Comment:
I'm also told that `#build-only` could also suit (I'm leaning towards `#fallback`)

Reply:
I think `#fallback` is good (similar to `<ClientOnly>`)